cypress/e2e/po/components/workloads/pod.po.ts

```diff
@@ -26,4 +32,16 @@ export default class PodPo extends CreateEditViewPo {
 
     shell.openTerminal();
   }
+
+  containerButton() {
+    return new PodContainer();
+  }
+
+  name(): LabeledInputPo {
+    return LabeledInputPo.byLabel(this.self(), 'Name');
+  }
+
+  createButton() {
+    return this.self().find('[data-testid="form-save"]');
+  }
```

Comment:
these should be

```
  nameNsDescription() {
    return new NameNsDescription(this.self());
  }

  saveCreateForm(): ResourceDetailPo {
    return new ResourceDetailPo(this.self());
  }
```

Reply:
when you drill down further to `cypress/e2e/po/edit/resource-detail.po.ts` it is the `AsyncButtonPo ` that is used